.build-tools/builtin-authentication-profiles.yaml

```diff
@@ -25,8 +25,31 @@ aws:
         description: |
           AWS session token to use. A session token is only required if you are using
           temporary security credentials.
+  - title: "AWS: Assume specific IAM Role"
+    description: |
+      Assume a specific IAM role. Note: This is only supported on Kafka and PostgreSQL components.
+    metadata:
+      - name: sessionToken
```

Comment:
sessionToken does not belong to this auth profile, sessionToken is used in combination with access key and secret key